plugins/modules/foreman_hostgroup.py

```diff
@@ -370,7 +333,9 @@ def main():
             entity_dict['content_view'] = module.find_resource_by_name('content_views', name=entity_dict['content_view'],
                                                                        params=scope, failsafe=False, thin=True)
 
-    entity_dict = module.handle_taxonomy_params(entity_dict)
+        entity_dict = module.handle_common_host_spec(entity_dict)
```

Comment:
the taxonomy handler has `if not self.desired_absent`, so you don't *have* to move it inside here, but I wonder if we should standardize on doing the if or not in the handler.

Reply:
Oh right. It was not called `maybe_handle_taxonomy_params` so i just assumed. ;)

I'm really not sure, which one is better.

Reply:
I slightly prefer to have the "if" in the handler, as this allows some modules to omit the if in their own code completely (those which don't reference anything but taxonomies).